.travis.yml

```diff
@@ -0,0 +1,23 @@
+os: 
+ - linux
+# we can't currently register URIs without a bundle and this project
```

Comment:
We should not be commenting out OSX - it should at-least be tested for compilation and then later we can see what else needs to be done for actually running the tests/examples.